features/zoek.feature

```diff
@@ -0,0 +1,56 @@
+#language: nl
+
+@post-assert
+Functionaliteit: Zoeken
+
+Rule: Er moet een valide zoek type worden opgegeven
+
+  @fout-case
+  Scenario: zoek zonder opgeven van type
+    Als personen wordt gezocht met de volgende parameters
+    | naam | waarde |
```

Comment:
is het hier niet interessant om wel andere parameters op te nemen? bijvoorbeeld wel postcode en huisnummer, maar niet type. dat is een ander testgeval/voorbeeld dan deze, want hier gaat het om een volledig lege request body

Reply:
scenario titel aangepast en nieuwe scenario `zoek zonder opgeven van 'type' parameter` toegevoegd